core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -160,6 +159,11 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
       azureRelayCreateResult <- traceWithParent("createAzureRelayInWSM", parentSpan)(_ =>
         Future(workspaceManagerDAO.createAzureRelay(workspaceId, workspaceRequest.region, userInfo.accessToken))
       )
+      // Create storage account before polling on relay because it takes ~45 seconds to create a relay
```

Comment:
I'm curious, how long does it take to create the storage account?

Reply:
I didn't time it, but my gut feeling was about 5 seconds. It won't increase the total time for workspace creation though since I did it before polling for the relay creation to complete.